docs/UWP/frameworks/caliburnmicro.md

```diff
@@ -1,5 +1,7 @@
 ﻿# Caliburn.Micro
 
+**Notice: Caliburn.Micro is no longer actively developed or maintained and will be removed as an option in a future version of Windows Template Studio.**
```

Comment:
Think we should refrase this as there seem to be new maintainers for Caliburn.Micro.